picasso/src/main/java/com/squareup/picasso3/Dispatcher.kt

```diff
@@ -63,10 +64,10 @@ internal class Dispatcher internal constructor(
   internal val hunterMap = mutableMapOf<String, BitmapHunter>()
 
   @get:JvmName("-failedActions")
-  internal val failedActions = mutableMapOf<Any, Action>()
+  internal val failedActions = WeakHashMap<Any, Action>()
 
   @get:JvmName("-pausedActions")
-  internal val pausedActions = mutableMapOf<Any, Action>()
+  internal val pausedActions = WeakHashMap<Any, Action>()
```

Comment:
This does result in using the JVM types again, which was probably the thing we were mistakenly optimizing for in the conversion.

Reply:
Yeah 😞 I wish Kotlin had a "first-party" equivalent